content/en/security/cloud_siem/security_operational_metrics.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Security Operational Metrics
+disable_toc: false
+further_reading:
+- link: "security/cloud_siem/investigate_security_signals"
+  tag: "Documentation"
+  text: "Investigate Cloud SIEM Security Signals"
+- link: "getting_started/dashboards"
+  tag: "Documentation"
+  text: "Getting started with dashboards"
+- link: "getting_started/monitors"
+  tag: "Documentation"
+  text: "Getting started with monitors"
+- link: "metrics/summary/"
+  tag: "Documentation"
+  text: "Learn more about the Metrics Summary"
+---
+
+## Overview
+
+Cloud SIEM provides security operational metrics to help you determine the effectiveness of your team in responding to and resolving security threats to your cloud environments. These metrics are shown in the out-of-the-box [Cloud SIEM dashboard][1] and also sent in the Cloud SIEM [weekly digest reports][2]. You can also create dashboards and monitors for them.
+
+{{< img src="security/security_monitoring/security_operational_metrics.png" alt="The security operational metrics section of the Cloud SIEM Overview dashboard" style="width:100%;" >}}
+
+## Operational Metrics
+
+`datadog.security.siem_signal.time_to_detect`
+: **Name**: Mean Time to Detect (MTTD)
+: **Description**:The average time from when a matching log is triggered to when a signal is generated.
+
+`datadog.security.siem_signal.time_to_acknowledge`
+: **Name**:  Mean Time to Acknowledge (MTTA)
+: **Description**: The average time from when a signal is triggered to when an investigation on the signal begins.
+
+`datadog.security.siem_signal.time_to_resolve`
+: **Name**: Mean Time to Resolve (MTTR)
```

Comment:
Can we replace all of the "Mean Time" to just "Time". We'd like to refer to the metrics as time to detect, time to acknowledge, etc. 

Mean can be derived from the metric itself.

Reply:
Removed `Mean`.